doc/integration/integration-prometheus-grafana.md

```diff
@@ -0,0 +1,176 @@
+# StarRocks Cluster Integration With Prometheus and Grafana Service
+
+This document describes how to integrate StarRocks cluster with Prometheus and Grafana service in a kubernetes environment. From this document, you will learn,
+* How to turn on prometheus metrics scrape for the StarRocks cluster
+* How to import StarRocks Grafana dashboard
+
+
+## 1. Prerequisites
+
++ Kubernetes: v1.18+ - v1.26
++ Prometheus helm chart: latest available version
++ Grafana helm chart: latest available version
++ StarRocks operator and helm chart: v1.7.1+
```

Comment:
please use helm chart v1.8.0+, becase the values.yaml is not compatible with previous versions

Reply:
1.8.0 is not released yet :)